packages/sources/web-components/src/components/vtmn-toggle/vtmn-toggle.tsx

```diff
@@ -0,0 +1,51 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'vtmn-toggle',
+  styleUrl: 'vtmn-toggle.pcss',
+  shadow: true,
+})
+export class VtmnToggle {
+  /**
+   * The id of the toggle.
+   * @default 'my-id'
+   */
+  @Prop() id: string = 'my-id';
```

Comment:
For me it's required (as you saif in story), because if not, we will have several toggles with the same id in a HTML page... So you can delete the `@default` value and the `= 'my-id';`.